unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -8,6 +8,12 @@ namespace DCL.Backpack
 {
     public class BackpackEditorHUDController
```

Comment:
I would like to warn you that this class is getting bigger and it's already not clear which responsibilities it has. You can consider splitting it into individual parts (represented by a `struct`) and passing `BackpackEditorHUDModel` to their constructor. So every part can handle its own piece of logic.

Reply:
Thanks for the feedback Mikhail! Since now we're all modifying this controller in a parallel way, in order to avoid conflicts, we're going to wait for a next step (maybe after we have added all the current ongoing functionalities) to apply that splitting 👍 But yeah, ansolutely, we will take it into account before releasing the feature.